nodes/put-object-acl/put-object-acl.html

```diff
@@ -0,0 +1,79 @@
+<!-- BEGIN PUT OBJECT ACL -->
+<script type="text/x-red" data-template-name="Put Object ACL">
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> <span data-i18n="Name">&nbsp;</span>Name: </label>
+        <input type="text" id="node-input-name" data-i18n="[placeholder]Name">
+    </div>
+    <div class="form-row">
+        <label for="node-input-conf"><code><i class="fa fa-cogs"></i></code> <span data-i18n="Configuration">&nbsp;</span>Client: </label>
+        <input type="text" id="node-input-conf">
+    </div>
+    <div class="form-row">
+        <label for="node-input-bucket"><code><i class="fa fa-database"></i></code> <span data-i18n="Bucket">&nbsp;</span>Bucket: </label>
+        <input type="text" id="node-input-bucket" data-i18n="[placeholder]Bucket">
+    </div>
+    <div class="form-row">
+        <label for="node-input-key"><code><i class="fa fa-magic"></i></code> <span data-i18n="Key">&nbsp;</span>Key: </label>
+        <input type="text" id="node-input-key" data-i18n="[placeholder]Object Key">
+    </div
+    <div class="form-row">
+        <label for="node-input-acl"><i class="fa fa-lock"></i> <span data-i18n="acl">&nbsp;</span>ACL: </label>
+        <select id="node-input-acl">
+            <option value="" selected>Choose ACL permissions</option>
+            <option value="private">Private</option>
+            <option value="public-read">Public Read</option>
+            <option value="public-read-write">Public Read Write</option>
+            <option value="authenticated-read">Authenticated Read</option>
+            <option value="aws-exec-read">AWS Exec Read</option>
+            <option value="bucket-owner-read">Bucker Owner Read</option>
+            <option value="bucket-owner-full-control">Bucket Owner Full Control</option>
+        </select>
+    </div>
+    <div class="form-tips">
+        <span>Tips:</span>
+        <ul>
+            <li><span data-i18n="[html]tip1">Leave <code>Bucket</code> | <code>Key</code> | <code>Source Bucket</code> | <code>Source Object</code> | <code>Destination Content Encoding</code> field(s) blank if you are setting it via msg property. <code><b><u>Red</u></b></code> iconed properites are <code>required!</code> E.g.:
```

Comment:
```suggestion
            <li><span data-i18n="[html]tip1">Leave <code>Bucket</code> | <code>Key</code> | <code>ACL</code> field(s) blank if you are setting it via msg property. <code><b><u>Red</u></b></code> iconed properites are <code>required!</code> E.g.:
```
Small "docs" update to correspond with **_`Put Object ACL`_** node